Mathlib/Data/Rat/Defs.lean

```diff
@@ -214,6 +214,14 @@ instance instPowNat : Pow ℚ ℕ where
   pow q n := ⟨q.num ^ n, q.den ^ n, by simp [Nat.pow_eq_zero], by
     rw [Int.natAbs_pow]; exact q.reduced.pow _ _⟩
 
+def pow_int_exponent (q : ℚ) (n : ℤ) : ℚ :=
+  if n ≥ 0
+  then q ^ (Int.toNat n)
+  else q ^ (Int.toNat (-n))
```

Comment:
Didn't you mean to add a `1/` here?

Either way, your `instPowInt` instance below already exists in mathlib; you can find it with
```lean
import Mathlib
#synth Pow ℚ ℤ
```

Reply:
(the name of the instance that it finds is bad though; fixed in #19884)